production-deployment-using-tiup.md

```diff
@@ -353,15 +353,47 @@ tiup cluster display tidb-test
 
 Expected output includes the instance ID, role, host, listening port, and status (because the cluster is not started yet, so the status is `Down`/`inactive`), and directory information.
 
-## Step 7: Start the TiDB cluster
+## Step 7: Start a TiDB cluster
+
+Safe start is newly introduced to TiUP cluster v1.9.0 and later versions. Logging into a database in this mode improves the security of the database. It is recommended that you start a TiDB cluster in this mode.
```

Comment:
```suggestion
Since TiUP cluster v1.9.0, safe start is introduced as a new start method. Logging into a database using this method improves the security of the database. It is recommended that you start a TiDB cluster using this method.
```